eidangservices/federator/server/request.py

```diff
@@ -180,6 +182,7 @@ class GranularFdsnRequestHandler(RequestHandlerBase):
     """
     Representation of a FDSN webservice request handler.
     """
+    HEADERS = {"user-agent": "EIDA-Federator/" + __version__}
```

Comment:
@Jollyfant, since you already added `HEADERS` to `RequestHandlerBase` this line is unnecessary.

Reply:
Actually I missed the inheritance and added them to the derived classes.. now I put them in the base class. Nice catch.